deploy/libproject/deploy.rb

```diff
@@ -268,6 +268,16 @@ def deploy(cmd_name, args)
       --quiet
   }
   maybe_log_jira.call "'#{op.opts.project}': completed UI service deployment"
+  common.run_inline %W{
+    ../public-ui/project.rb deploy-ui
+      --project #{op.opts.project}
+      --account #{op.opts.account}
+      --key-file #{op.opts.key_file}
+      --version #{op.opts.app_version}
+                    #{op.opts.promote ? "--promote" : "--no-promote"}
```

Comment:
nit: unindent this

Reply:
I've tried to unindent this a bunch. Got it this time. IntelliJ is weird sometimes with autoindent and just doesn't play nice.  I just had to use spaces ..